### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by domsinclair**

### CODE REVIEW

**Comment by biohazard999** [comment]
Looks good to me!
Thanks for the contrib <3

File: docs/guide/README.md [hunk 1]
```diff
@@ -4,29 +4,29 @@ title: Introduction
...
+Xenial.Framework has been designed to be:
 
 #### Easy
 
 You don't need to learn any new concepts if you are already familiar with XAF, no matter what skill level you are at.
 
 #### Flexible
 
-We provide solutions at any level of the stack, you can use the *whole* package, but your are not forced to.
+Use as little or as much of the Xenial.Framework as is relevant to your current application and workflow.
```
Comment (biohazard999) [inline docs/guide/README.md:18]: We probably should expand on this idea, but its hard to get it into one sentence, maybe we should link to sub pages, but I don't want to overcomplicate it. But your wording makes a lot more sense.

File: docs/guide/README.md [hunk 2]
```diff
@@ -4,29 +4,29 @@ title: Introduction
...
 
 #### Focused
 
-Features are developed with years of experience, solving most (LOB) development challenges at it's core.
+Practical solutions to the problems and challenges experienced from years of working on Line of Business (LOB) applications are at the core of the Xenial.Framework.
```
Comment (biohazard999) [inline docs/guide/README.md:22]: Love that one!

File: docs/guide/README.md [hunk 3]
```diff
@@ -4,29 +4,29 @@ title: Introduction
...
 
 ## Key Concepts
 
-Xenial.Framework supports both the [ModelEditor](https://docs.devexpress.com/eXpressAppFramework/112582/concepts/application-model/model-editor) and code-centric approaches and uses all the basic primitives that XAF provides. We also provide a convention based framework that helps you and your team to create your own domain specific rules, resulting in more consistent applications with a less configuration heavy approach.
+Xenial.Framework supports both the [ModelEditor](https://docs.devexpress.com/eXpressAppFramework/112582/concepts/application-model/model-editor) and code-centric approaches utilising all the basic primitives that XAF provides. It supports a convention based framework that enables individuals or teams to create their own domain specific rules, resulting in more consistent applications requiring less configuration.
 
-::: tip
-You can mix and match all approaches and use the right tool for a particular problem
+::: tip INFORMATION
+Xenial.Framework actively promotes a mixed approach to development, promoting the paradigm of the right tool for the job.
```
Comment (biohazard999) [inline docs/guide/README.md:29]: Maybe we should engage the user by mention that he does not need to change his XAF workflow, so he can drop in.

File: docs/guide/README.md [hunk 4]
```diff
@@ -4,29 +4,29 @@ title: Introduction
 
 # Introduction
 
-**Xenial.Framework** is a set of powerful modules for [DevExpress expressAppFramework](https://www.devexpress.com/XAF).  
-Handcrafted after 10 years experience with the framework, from small applications to large scale systems with dozens of developers, it helps you build robust, state of the art applications quickly and allow you to scale your future development efforts easily, without getting in your way.
+**Xenial.Framework** is a set of powerful modules for the [DevExpress expressAppFramework (XAF)](https://www.devexpress.com/XAF).  
+Created ater more that ten years working experience with XAF, from small applications to large scale systems with dozens of developers, the Xenial Framework helps you build robust, state of the art applications quickly and allows you to scale your future development efforts easily, without getting in your way.
```
Comment (Paramethod) [inline docs/guide/README.md:8]: ater -> after

Comment (biohazard999) [reply docs/guide/README.md:8]: 👍

File: docs/guide/README.md (hunk 1 above)
Comment (biohazard999) [reply docs/guide/README.md:18]: maybe add `, we reduce any overhead to a minumum` ? But i think its a lot clearer than before

File: docs/guide/README.md (hunk 4 above)
Comment (domsinclair) [reply docs/guide/README.md:8]: after is missing an f, one day I'll actuall learn how to type.

**Pull request merged**